streams/src/test/java/org/apache/kafka/streams/processor/internals/AssignedStreamsTasksTest.java

```diff
@@ -571,5 +539,16 @@ private void mockRunningTaskSuspension() {
         EasyMock.expectLastCall();
     }
 
+    private void verifyTaskMigratedExceptionDoesNotCloseTask(final ThrowingRunnable action) {
+        final Set<TaskId> expectedRunningTaskIds = Collections.singleton(taskId1);
+
+        // This action is expected to throw a TaskMigratedException
+        assertThrows(TaskMigratedException.class, action);
+
+        // This task should be closed as a zombie with all the other tasks during onPartitionsLost
+        assertThat(assignedTasks.runningTaskIds(), equalTo(expectedRunningTaskIds));
+
+        EasyMock.verify(t1);
+    }
 
 }
```

Comment:
nit: empty line

Reply:
You mean right before the end of the file? A lot of them seem to have that, not sure if intentional or not though